about.md

```diff
@@ -94,3 +94,25 @@ DELETE https://<host>:<port>/<index-name>/_doc/<document-id>
 ```
 
 You can change most OpenSearch settings using the REST API, modify indexes, check the health of the cluster, get statistics---almost everything.
+
+## Advanced concepts
+
+The following section describes more advanced OpenSearch concepts.
+
+### Translog
+
+Any index changes, such as indexing or deleting documents, are written to disk during a Lucene commit. However, Lucene commits are expensive operations, so they cannot be performed after every change to the index. Instead, each shard records every indexing operation in a transaction log called _translog_. When a document is indexed, it is added to the memory buffer and recorded in the translog. After a process or host restart, any data in the in-memory buffer is lost. Recording the document in the translog ensures durability because the translog is written to disk.
+
+Frequent refresh operations write the documents in the memory buffer to a segment and then clear the memory buffer. Periodically, a [flush](#flush) performs a Lucene commit, which includes writing the segments to disk using `fsync`, purging the old translog, and starting a new translog. Thus, a translog contains all operations that have not yet been flushed.
+
+### Refresh
+
+Periodically, OpenSearch performs a _refresh_ operation, which writes the documents from the in-memory Lucene index to files. These files are not durable because they reside in the file cache. A refresh makes documents available for search.
```

Comment:
```suggestion
Periodically, OpenSearch performs a _refresh_ operation, which writes the documents from the in-memory Lucene index to files. These files are not guaranteed to be durable because an `fsync` is not performed. A refresh makes documents available for search.
```